packages/data-transport-layer/src/services/l1-ingestion/handlers/sequencer-batch-appended.ts

```diff
@@ -247,7 +249,7 @@ const maybeDecodeSequencerBatchTransaction = (
       target: toHexString(decodedTx.to), // Maybe null this out for creations?
       data: toHexString(decodedTx.data),
       sig: {
-        v: BigNumber.from(decodedTx.v).toNumber(),
+        v: BigNumber.from(decodedTx.v).toNumber() - 2 * l2ChainId - 35,
```

Comment:
nit: would put this into a function so it's more readable what's being done here